apps/lockfile-explorer-web/webpack.config.js

```diff
@@ -17,11 +17,11 @@ module.exports = function createConfig(env, argv) {
         }
       },
       performance: {
-        hints: env.production ? 'error' : false
+        hints: env.production ? 'error' : false,
         // This specifies the bundle size limit that will trigger Webpack's warning saying:
         // "The following entrypoint(s) combined asset size exceeds the recommended limit."
-        // maxEntrypointSize: 500000,
-        // maxAssetSize: 500000
+        maxEntrypointSize: Infinity,
```

Comment:
If you're setting these to infinity, why not just turn off performance?

Reply:
Is there a better way to do that? I tried `undefined` but in the merge it doesn't undo the rig's setting.

Reply:
It's `performance: false`